### ISSUE TITLE
[BUG] `sslcert` doesn't support any `state`

### ISSUE THREAD

**Issue by fa-elepape** [netscaler/ansible-collection-netscaleradc#295]
**Describe the bug**
When using `netscaler.adc.sslcert`, the module fails at the ansible module parameters validation because no `state` is supported:

```json
{
    "changed": false,
    "msg": "value of state must be one of: , got: present"
}
```

**To Reproduce**
Steps to reproduce the behaviour:

1. My ansible-playbook is
```yaml
---
- name: netscaler.adc.sslcert bug report
  hosts: localhost
  gather_facts: false

  tasks:
  - netscaler.adc.sslcert:
      state: present
```

2. The ansible-playbook command I executed is...
```bash
$ ansible-playbook --diff bug-report.yml
PLAY [netscaler.adc.sslcert bug report] **************************************************************************

TASK [netscaler.adc.sslcert] *************************************************************************************
fatal: [localhost]: FAILED! => {"changed": false, "msg": "value of state must be one of: , got: present"}

PLAY RECAP *******************************************************************************************************
localhost                  : ok=0    changed=0    unreachable=0    failed=1    skipped=0    rescued=0    ignored=0
```

**Expected behaviour**
The `sslcert` creation succeeds.

**Environment (please complete the following information):**
```bash
$ ansible-galaxy collection list netscaler.adc
Collection    Version
------------- -------
netscaler.adc 2.0.1

$ python3 --version
Python 3.9.16

$ ansible --version
ansible [core 2.13.4]
  config file = /etc/ansible/ansible.cfg
  configured module search path = ['~/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/local/share/ansible/lib64/python3.9/site-packages/ansible
  ansible collection location = /etc/ansible/collections:/usr/share/ansible/collections
  executable location = /usr/local/bin/ansible
  python version = 3.9.16 (main, May 29 2023, 00:00:00) [GCC 11.3.1 20221121 (Red Hat 11.3.1-4)]
  jinja version = 3.1.2
  libyaml = True
```

**Additional context**
The [`_supported_operations`](../blob/main/plugins/module_utils/nitro_resource_map.py#L32550) of the `sslcert` do not include the `add` or `update` expected by the [`get_valid_desired_states`](../blob/main/plugins/module_utils/common.py#L441) function.

**Comment by sumanth-lingappa** [comment]
@fa-elepape, thank you for raising this issue.
I acknowledge the bug in the collection for `sslcert` in specific as well as all the resources with `create` as action in the NITRO URL.

We will keep this in our planning and update here as it is fixed.

**Comment by sumanth-lingappa** [comment]
This issue is fixed in the latest release `2.1.0`. Please check and update the issue accordingly.

**Comment by sumanth-lingappa** [comment]
closing this. please update this request if you still face the issue.

Refer below for an example

https://github.com/netscaler/ansible-collection-netscaleradc/blob/7eee55f1677a75f8eaca085cd3ad498be68791ed/examples/sslcert.yaml#L9
